.eslintignore

```diff
@@ -1,3 +1 @@
-dist
-lib
-test
+bundle.js
```

Comment:
Everything that is in gitignore file also is ignored by eslint. Maybe this file is redundant.